Feature: add footer message

luigirosa opened the pull request:
Adds an optional footer message to display warning, disclaimers and company policies.

luigirosa committed:
Add footer message

Add optional footer message

GaryAllan reviewed (changes requested):
Hi @luigirosa

Could you move down to the general tweaks section and default to off?
Also update the footer check to:

```php
# custom footer message
if(is_string($config['footer_message']) && strlen($config['footer_message'])>0) {
```

Users will copy config.dist.php to config.php on first install so the default should be to show nothing.

``` php
/**
 * General tweaks
 ******************************/
$config['logo_width']             = 220;                    // logo width
$config['requests_public']        = true;                   // Show IP request module on login page
$config['split_ip_custom_fields'] = false;                  // Show custom fields in separate table when editing IP address
$config['footer_message']         = "";                     // Custom message included in the footer of every page
```

luigirosa committed:
Footer message default eand location

$config['footer_message'] default to empty string and moved to General tweaks section

luigirosa committed:
Better check of $config['footer_message']
